packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -96,17 +96,19 @@ export async function saveProjectIdToAppConfigAsync(
     case 'success':
       break;
     case 'warn': {
-      Log.log();
+      Log.warn();
       Log.warn('It looks like you are using a dynamic configuration!');
-      Log.log(
+      Log.warn(
         chalk.dim(
           'https://docs.expo.dev/workflow/configuration/#dynamic-configuration-with-appconfigjs)\n'
         )
       );
       Log.warn(
-        'In order to finish setting up your project you are going to need manually add the following to your "extra" key:\n\n'
+        'In order to finish setting up your project you are going to need manually edit app.json/app.config.js and add the following to your "extra" key:'
```

Comment:
```suggestion
        'To complete the setup process, set the `${chalk.bold('eas.extra.projectId')} in your `${chalk.bold('app.config.js')}`:'
```

replace app.config.js with the actual file name as discussed above